sympy/combinatorics/rewritingsystem_fsm.py

```diff
@@ -27,39 +28,41 @@ def __init__(self, name, is_start=False, is_dead=False, is_accept=False, subst=N
             self.state_type = "dead"
 
     def add_transition(self, letter, state):
-        """
-        This method is to add a transition from the current state to a new state.
+        '''
+        Add a transition from the current state to a new state.
 
-        Arguments:
-            letter: The alphabet element the current state reads to make the state transition.
-            state: This will be an instance of the State object which represents a new state after in the transition after the alphabet is read.
-        """
+        Keyword Arguments:
+            letter -- The alphabet element the current state reads to make the state transition.
+            state -- This will be an instance of the State object which represents a new state after in the transition after the alphabet is read.
+
+        '''
         self.transitions[letter] = state
 
 class StateMachine(object):
-    """
-    The state machine class which manages the states and their transisitons of the automata.
+    '''
+    Represents the attributes and methods of a finite state machine.
```

Comment:
I think I actually preferred the previous version here because docstrings for classes in SymPy are often nouns. You could also say something like `A representation/implementation of a finite state macine that manages ...`. Well done with the other docstrings though, they look quite good.